php/5.6/node/8/scripts/docker-as-symfony

```diff
@@ -0,0 +1,440 @@
+
+  # Populate mailer config
+  sed -ri -e "s@mailer_host:.+@mailer_host: ${SMTP_HOST%:*}@g" /var/www/app/config/parameters.yml
+
+  # Fix rights on cache directores
+  mkdir -p /var/www/app/cache
+  chmod -R 777 /var/www/app/cache
+  find /var/www/app/cache -type f -executable -exec chmod -x {} \;
+  find /var/www/app/cache -type d -exec chmod +xs {} \;
+
+  # Fix right on logs directory
+  mkdir -p /var/www/app/logs
+  chmod -R 777 /var/www/app/logs
+  find /var/www/app/logs -type f -executable -exec chmod -x {} \;
+  find /var/www/app/logs -type d -exec chmod +xs {} \;
+
+  php -d memory_limit=-1 app/console cache:clear --no-interaction
+}
+
+# Return TRUE if Drupal is already installed (database is ready)
+function symfony_is_installed() {
+  if [ $SYMFONY_IS_INSTALLED -eq 0 ]; then
+   # TODO: To be impletemed for symfony using migration table
+   # SYMFONY_IS_INSTALLED=$(app/console doctrine:query:sql "SELECT * FROM users LIMIT 1" >/dev/null 2>&1 && echo 1 || echo 0)
+   SYMFONY_IS_INSTALLED=1
+  fi
+
+  [ $SYMFONY_IS_INSTALLED -eq 1 ]
+}
+
+# Return FALSE if Drupal is not already installed (database is not ready)
+function symfony_not_installed() {
+  ! symfony_is_installed
+}
+
+# Return TRUE if yarn is installed
+function yarn_is_installed() {
+  command -v yarn > /dev/null 2>&1
+}
+
+# Enable fake time
+function lock_time() {
+  if [ ! -z "$LOCK_TIME" ]; then
+    export LD_PRELOAD="/usr/local/lib/faketime/libfaketime.so.1"
+    export FAKETIME="${LOCK_TIME:-$1}"
+  fi
+}
+
+# Disable fake time
+function free_time() {
+  unset LD_PRELOAD
+}
+
+# Install dependencies
+function install_dependencies() {
+  printf "\e[1;35m* Install dependencies.\e[0m\n"
+
+  composer install --prefer-dist --no-progress --no-suggest --no-interaction
+  if yarn_is_installed && [ -f /var/www/package.json ] && [ -f /var/www/yarn.lock ]; then
+    yarn install --no-progress --non-interactive
+  fi
+}
+
+# Reset database with dump made before loading default content on bootstrap.
+function database_reset() {
+  FILE_PATH=${1:-"$DATABASE_DUMP"}
+
+  if [ -f $FILE_PATH ]; then
+    printf "\e[1;35m* Reset database.\e[0m\n"
+    if symfony_not_installed; then
+      printf "\e[1;35m* (maybe forced beceause not installed).\e[0m\n"
+    fi
+  elif [ "$FILE_PATH" != "$DATABASE_DUMP" ]; then
+    printf "\e[01;31No database dump found, run '$SCRIPT_NAME db-dump' first.\e[0m\n"
+    return 1
+  else
+    printf "\e[01;31No database dump found, run '$SCRIPT_NAME bootstrap' first.\e[0m\n"
+    return 1
+  fi
+
+  # Drop and create database
+  app/console doctrine:database:drop --no-interaction --quiet --if-exists --force
+  app/console doctrine:database:create --no-interaction --quiet
+
+  # Load database dump to reset its state
+  mysql --user="$DATABASE_USER" --password="$DATABASE_PASSWORD" --host="$DATABASE_HOST" $DATABASE_NAME < $FILE_PATH
+  app/console doctrine:schema:update --no-interaction --quiet --force
```

Comment:
I prefer `doctrine:migrations:migrate` (if doctrine migration installed) or nothing at all.

Reply:
If migrate is installed, I will use migrate, but for a dev environment, when you switch branch you want to be able to reset your database properly, or to load a dump and be sure is up to date with your current schema. 

It's the purpose of this function, so if we don't do that, we will have issues on Behat tests per example.

I use mysql load, then update because we can use this to load a dump (from production), if that was used only to reset database to empty state, `doctrine:create` could be used. But the `db-restore` need to ensure that schema is up to date.

Bu if doctrine migration can be install for Symfony 2.8, we can force to use it.

Reply:
done